common/lib/permissions.js

```diff
@@ -25,6 +25,18 @@ const policePermissions = [
   'person_escort_record:confirm',
 ]
 
+const contractDeliveryManagerPermissions = [
+  'allocations:view',
```

Comment:
Do we want to give them `dashboard:view` access too?

Reply:
The ticket says just moves and stuff for now, and dashboard later, and I suspect we will be tweaking these permissions when people actually have a play anyway. I think I'll merge it like this, have product have a play and then update with whatever is missing.